source/whats-new.txt

```diff
@@ -54,6 +54,10 @@ The 5.0 driver release introduces the following features:
   name, or it defaults to ``MULTIPLE`` if you specify more than one
   host.
 
+- The ``ConnectionCheckedOutEvent.getElapsedTime`` method now includes the time
+  taken to create the connection. That is, the "timer" starts before the
+  connection is confirmed, rather than after.
+
```

Comment:
I think more technical detail would be helpful here. I'd probably just paste in the Jira notes and maybe edit them slightly.

Also, I think the ticket describes more changes than I see here; is there another ticket for those?